packages/makedumpfile/makedumpfile.spec

```diff
@@ -1,5 +1,5 @@
 Name: %{_cross_os}makedumpfile
-Version: 1.7.2
+Version: 1.7.3
```

Comment:
Added to the lookaside cache, but looks like the new version includes more files that need to be handled in the `%files` section.

```
  #18 7.714 error: Installed (but unpackaged) file(s) found:
  #18 7.714    /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/makedumpfile/eppic_scripts/README
  #18 7.714    /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/makedumpfile/eppic_scripts/ap_messages_3_10_to_4_8.c
  #18 7.714    /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/makedumpfile/eppic_scripts/dir_names_3_10_to_3_13.c
  #18 7.714    /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/makedumpfile/eppic_scripts/dir_names_3_14_to_4_8.c
  #18 7.714    /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/makedumpfile/eppic_scripts/keyring_3_10_to_4_3.c
  #18 7.714    /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/makedumpfile/eppic_scripts/keyring_4_4_to_4_8.c
  #18 7.714    /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/makedumpfile/eppic_scripts/proc_names_3_10_to_4_8.c
  #18 7.714    /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/makedumpfile/eppic_scripts/tcp_sk_buf_3_10_to_4_8.c
  #18 7.714    /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/makedumpfile/eppic_scripts/udp_sk_buf_3_10_to_4_8.c
  #18 7.714    /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/makedumpfile/eppic_scripts/unix_sk_buff_3_10_to_4_8.c
  #18 7.714    /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/makedumpfile/eppic_scripts/vhost_net_buffers_3_10_to_3_18.c
  #18 7.714    /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/makedumpfile/eppic_scripts/vhost_net_buffers_3_19_to_4_8.c
  #18 7.714    /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/makedumpfile/eppic_scripts/vhost_scsi_buffers_3_10_to_4_8.c
  #18 7.714    /x86_64-bottlerocket-linux-gnu/sys-root/usr/share/makedumpfile/makedumpfile.conf.sample
```

Reply:
I think we just need to ignore these two entries: https://github.com/makedumpfile/makedumpfile/blob/ecc9355e1344d3de94870e855e380ee84224e552/makedumpfile.spec#L41-L42

Reply:
Fixed this by changing spec file 👍